.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -1,3 +1,8 @@
+---
+name: Bug Report 🐞
+about: Something isn't working as expected? Here is the right place to report.
+---
```

Comment:
everything above the `---` in this PR came directly from the templates we use in `LightGBM` (https://github.com/microsoft/LightGBM/tree/master/.github/ISSUE_TEMPLATE)